src/app/modules/item/pages/chapter-user-progress/chapter-user-progress.component.html

```diff
@@ -47,22 +47,49 @@
           <td *ngFor="let col of columns">
             <ng-container [ngSwitch]="col.field">
               <ng-container *ngSwitchCase="'title'">
-                <a
-                  class="alg-link"
-                  [ngClass]="{'disabled': rowData.id === itemData?.item?.id}"
-                  (click)="onClick(rowData)"
-                >
-                  {{ rowData.title }}
-                </a>
+                <div class="title">
+                  <a
+                    class="alg-link"
+                    [ngClass]="{'disabled': rowData.id === itemData?.item?.id}"
+                    (click)="onClick(rowData)"
+                  >
+                    {{ rowData.title }}
+                  </a>
+
+                  <ng-container *ngIf="rowData.type === 'Task' && rowData.currentUserPermissions">
+                    <button
+                      pButton
+                      class="alg-button p-button-rounded p-small-button p-button-outlined load-answer-button"
+                      [label]="(!!(watchedGroup$ | async) ? 'watchingGroup' : '') | i18nSelect : {
```

Comment:
You should not resubscribe for every row (see https://github.com/France-ioi/AlgoreaFrontend/blob/master/src/app/modules/item/pages/item-log-view/item-log-view.component.html#L21)

Reply:
Fixed